drivers/sony-bravia-android-tv/device.js

```diff
@@ -10,19 +9,34 @@ class SonyBraviaAndroidTvDevice extends Homey.Device {
 
     console.log(`${this.data.name} initialized.`);
 
-    this.registerTasks();
-    this.registerFlowListeners();
-    this.registerCapabilityListeners();
+    this.setCapabilityListeners();
+    this.checkDeviceInterval(this.data.settings.polling);
+
   }
 
   onDeleted() {
     this.data = this.generateDeviceObject();
 
     console.log(`${this.data.name} deleting.`);

```

Comment:
Should we replace all cases of `console.log` with `this.log` or is there a specific rule behind it?

Reply:
Indeed this.log() is recommendation and homey eslint is pushing to remove console.log.